.pre-commit-config.yaml

```
@@ -14,13 +14,13 @@ repos:
 
   # Autoformat: markdown, yaml
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.7.1
+    rev: v3.0.0-alpha.0
```

```suggestion
    rev: v2.7.1
```

Reverting the prettier bump as it is failing